policy/modules/services/zfs.fc

```diff
@@ -8,6 +8,7 @@
 /usr/sbin/zpool	--	gen_context(system_u:object_r:zfs_exec_t,s0)
 
 /etc/zfs(/.*)?		gen_context(system_u:object_r:zfs_config_t,s0)
+/etc/zfs/zfs-functions	--	gen_context(system_u:object_r:bin_t,s0)
```

Comment:
This should be in corecommands.fc instead.

Reply:
Makes sense. Done.